ui/src/authHelper.js

```diff
@@ -8,7 +8,9 @@ async function getToken() {
     console.error("Failed to fetch JWT token", error);
   }
 }
-
-module.exports = {
-  getToken,
-};
+function isTokenExpired(jwtToken) {
+  const decodedToken = JSON.parse(atob(jwtToken.split(".")[1]));
+  const currentTime = Math.floor(Date.now() / 1000);
+  return decodedToken.exp < currentTime;
+}
+export { getToken, isTokenExpired };
```

Comment:
Is this something you could add a unit test for?

Reply:
made test